smsgateway/utils.py

```diff
@@ -20,7 +22,15 @@ def check_cell_phone_number(number):
     return format_number(parsed_number, PhoneNumberFormat.E164)
 
 
-def truncate_sms(text, max_length=160):
+def get_max_msg_length():
+    account = get_account()
+    try:
```

Comment:
if this is to return something if there is the key or None otherwise, please use:
```
def get_max_msg_length():
    return get_account().get('max_msg_length')
```
It's one line, more readable and it is *a lot* cheaper as it does not rely on Exceptions.

It also works if you want to return something other than `None` otherwise: `my_dict.get(some_key, default_value)` (where `default_value` is `None` by default)